R/rvar-.R

```diff
@@ -546,10 +546,17 @@ copy_dimnames <- function(src, src_i, dst, dst_i) {
 # helpers: applying functions over rvars ----------------------------------
 
 # apply a summary function within each draw of the rvar (dropping other dimensions)
-summarise_rvar_within_draws <- function(x, .f, ...) {
+summarise_rvar_within_draws <- function(x, .f, ..., .transpose = FALSE, .when_empty = .f()) {
   draws <- draws_of(x)
   dim <- dim(draws)
-  new_rvar(apply(draws, 1, .f, ...), .nchains = nchains(x))
```

Comment:
I think that at least `.transpose` requires an input check via `as_one_logical`

Reply:
summarise_rvar_within_draws() is only called from internal code, and `.transpose` is not exposed to user input, so it should not be possible for it to be anything but a single logical value (in fact it's only used to implement rvar_range at the moment). So I would not expect the input check to ever fail. That said, if you prefer having the check there I will add it.

Reply:
Thanks, that make sense and is fine as is.